lib/workers/repository/onboarding/branch/commit-message.ts

```diff
@@ -0,0 +1,41 @@
+import { RenovateConfig } from '../../../../config/types';
+import { CommitMessage } from '../../model/commit-message';
+
+export class OnboardingCommitMessageFactory {
```

Comment:
Shouldn't this class be at same folder as commit-message?

Reply:
I would prefer to store it inside `onboarding` folder, because this class is specific only for onboarding functionality